backend/src/v4/models/risk.js

```diff
@@ -163,6 +175,15 @@ class Risk extends Ticket {
 			updatedRisk.data = {...updatedRisk.data, ...levelOfRisk};
 		}
 
+		try{
+			const teamspaceSettings = await TeamspaceSetting.getTeamspaceSettings(user,	{  _id: 0, createTreatmentSuggestions: 1});
```

Comment:
same here

If we do it in mitigations then you won't even have the duplicated code :)